autogpt_platform/frontend/src/lib/autogpt-server-api/index.ts

```diff
@@ -1,5 +1,6 @@
 import { AutoGPTServerAPI } from "./client";
 
+// TODO: rename to BackendAPI
```

Comment:
Why not do this in this PR?

Reply:
Out of scope, would cause a huge amount of changes and also conflicts on ongoing PRs